File: console_edition_Java/MyChess/Coord.java
package MyChess;

/**
 * 坐标类
 */
public class Coord {

    /** 行号 */
    public int x;
    /** 列号 */
    public int y;

    public Coord() {
        x = 0;
        y = 0;
    }

    /**
     * @param x 行号
     * @param y 列号
     */
    public Coord(int x, int y) {
        this.x = x;
        this.y = y;
    }

    @Override
    public String toString() {
        return "(" + x + ", " + y + ")";
    }
}

File: src/GameAlgo/Coord.js
/**
 * 坐标类
 */
export default class Coord {
    /**
     * @constructor
     * @param {Number} [x=0] 行号
     * @param {Number} [y=0] 列号
     * @param {Number} [h] 分数
     */
    constructor(x = 0, y = 0, h) {
        /**坐标行号 */
        this.x = x;
        /**坐标列号 */
        this.y = y;
        /**分数 */
        this.h = h;
    }
}